example/lib/custom_build_step.ex

```diff
@@ -0,0 +1,11 @@
+defmodule ExampleCliApp.CustomBuildStep do
+  alias Burrito.Builder.Step
+  @behaviour Step
+
+  def execute(% Burrito.Builder.Context{} = context) do
```

Comment:
Nitpick: this is missing an `@impl`

```suggestion
  @impl Step
  def execute(% Burrito.Builder.Context{} = context) do
```

Reply:
There's an odd space after the % sign, so you may want to run the formatter.